### PR TITLE
feat(bake/oracle): Added oracle OCI bakery

### PR DESCRIPTION
**Pull request by guoyongzhang**
Adding bakery capability to the oracle provider.
See also https://github.com/spinnaker/halyard/pull/1011, https://github.com/spinnaker/orca/pull/2405, and https://github.com/spinnaker/deck/pull/5697.

### CODE REVIEW

File: rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/oracle/OCIBakeHandler.groovy [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/*
```
Comment (robzienert) [inline rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/oracle/OCIBakeHandler.groovy]: Shout out to not including Groovy code in main src tree.

Comment (guoyongzhang) [reply rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/oracle/OCIBakeHandler.groovy]: What is recommended approach?  Place java code in rosco-core/src/main/groovy or create a new directory rosco-core/src/main/java?

Comment (robzienert) [reply rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/oracle/OCIBakeHandler.groovy]: Placing it in `groovy` is fine.

**Comment by guoyongzhang** [comment]
The PR has been updated to convert groovy code to Java code.  Please merge if review reveals no problem.

**Comment by guoyongzhang** [comment]
Could someone with merge privilege please merge this PR?

**Pull request merged**